sdcm/nemesis.py

```diff
@@ -6564,3 +6677,21 @@ class EndOfQuotaNemesis(Nemesis):
 
     def disrupt(self):
         self.disrupt_end_of_quota_nemesis()
+
+
+class IsolateNodeWithProcessSignalNemesis(Nemesis):
+    disruptive = True
+    topology_changes = True
+    kubernetes = False
+
+    def disrupt(self):
+        self.disrupt_refusing_connection_from_banned_node(use_iptables=False)
+
+
+class IsolateNodeWithIptableRuleNemesis(Nemesis):
```

Comment:
This option isn't gonna be used by any Nemesis

Only if this class is used directly.

disrupt function shouldn't have parameters, it causes the wrong expectations.

Reply:
create 2 different disrupt functions without parameter and switch particular classes to use them